tests/init/test_dynamic_api_spec.yaml

```
@@ -94,6 +94,9 @@ SciViz: # top level tab
                   destination: c_id
                 - type: attribute #omitted input means display is session_note
                   destination: c_name
+              preset_function: >
+                def preset_function():
+                    return {'preset 1': {'b_id': 14}}
```

Can we update this to the following? I think it is a bit cleaner.

```suggestion
              presets: >
                def presets():
                    return {'preset 1': {'b_id': 14}}
```